pkg/kubelet/metrics/metrics.go

```diff
@@ -79,6 +79,19 @@ const (
 	// Metrics keys for RuntimeClass
 	RunPodSandboxDurationKey = "run_podsandbox_duration_seconds"
 	RunPodSandboxErrorsKey   = "run_podsandbox_errors_total"
+
+	// Metrics to keep track of objects under management
+	ManagedPodsKey                  = "managed_pods"
+	ManagedContainersKey            = "managed_containers"
+	StartedPodsTotalKey             = "started_pods_total"
```

Comment:
We have kube_pod_start_time and kube_pod_info in kube-state-metrics, which can be summed into the total started pods. Note we have actual drill down of pod name. I guess if folks don't use KSM this can be useful and its not too much cardinality.

https://github.com/kubernetes/kube-state-metrics/blob/master/docs/pod-metrics.md